app.py

```diff
@@ -329,17 +331,23 @@ def list_data_objects(**kwargs):
     :rtype: ListDataObjectsResponse
     """
     req_body = app.current_request.query_params or {}
-    page_token = req_body.get('page_token', 0)
     per_page = int(req_body.get('page_size', 10))
-    if req_body.get('alias', None):
-        results = azul_match_alias(index=INDEXES['data_obj'],
-                                   alias=req_body['alias'], size=per_page + 1,
-                                   from_=page_token if page_token != 0 else None)
-    else:
-        results = es_query(query={}, index=INDEXES['data_obj'], size=per_page + 1)
+
+    # Build the query. If multiple criteria are specified, returned objects
+    # should match all of the provided criteria (logical AND).
+    query = {'query': {}}
+    if 'page_token' in req_body:
+        query['from'] = req_body['page_token'] or 0
```

Comment:
If this if statement doesn't trigger, does `query['from']` still default to zero?

Reply:
If there is no `page_token` parameter in the request body, `query['from']` does not need to be included